hubble-be/src/main/java/com/baidu/hugegraph/service/load/LoadTaskService.java

```diff
@@ -294,19 +295,25 @@ public void pauseAllTasks() {
      * Update progress periodically
      */
     @Async
-    @Scheduled(fixedDelay = 5 * 1000)
+    @Scheduled(fixedDelay = 1 * 1000)
     @Transactional(isolation = Isolation.READ_COMMITTED)
     public void updateLoadTaskProgress() {
-        Map<Integer, LoadTask> taskContainer = this.getTaskContainer();
-        Iterator<Map.Entry<Integer, LoadTask>> iter = taskContainer.entrySet()
-                                                                   .iterator();
+        Iterator<Map.Entry<Integer, LoadTask>> iter;
+        iter = this.taskContainer.entrySet().iterator();
         iter.forEachRemaining(entry -> {
             LoadTask task = entry.getValue();
-            LoadContext context = task.context();
-            task.setFileReadLines(context.newProgress().totalInputReaded());
-            task.setDuration(context.summary().totalTime());
-            if (this.update(task) != 1) {
-                throw new InternalException("entity.update.failed", task);
+            task.lock.lock();
+            try {
+                if (task.getStatus().inRunning()) {
+                    LoadContext context = task.context();
+                    task.setFileReadLines(context.newProgress().totalInputReaded());
+                    task.setCurrDuration(context.summary().totalTime());
+                    if (this.update(task) != 1) {
```

Comment:
what does 1 mean, prefer boolean or enum

Reply:
1 means operate affected rows. it defined by mybatis-plus framework.